frontend/lib/identification/card_detail_view/verification_qr_code_view.dart

```diff
@@ -33,7 +33,7 @@ class _VerificationQrCodeViewState extends State<VerificationQrCodeView> {
     _otpCode = widget._otpGenerator.generateOTP(_resetQrCode);
   }
 
-  _resetQrCode() {
+  void _resetQrCode() {
```

Comment:
I think this void is unnecessary. I would prefer not having to write void if this results in an error.

Reply:
We use this rule https://dart-lang.github.io/linter/lints/always_declare_return_types.html which enforces return types for all functions. Quite reasonable imo and it is probably not possible to exclude void. I can remember having the same discussion in Integreat :D But if you want we can disable the rule completely. @Schedulaar what do you think?

Reply:
I think this lint can be disabled. In my opinion it is very clear what this is doing:

```
_resetQrCode() {
    // ...
}
```

Writing the following instead does not add to the readability:

```
void _resetQrCode() {
    // ...
}
```

This is also the default in all modern languages like Rust, Kotlin, Dart etc.

Reply:
I'm fine with disabling it, just keep in mind that we don't have other return types on functions as well then.